cassandra/cqlengine/columns.py

```diff
@@ -72,6 +73,7 @@ def get_property(self):
             return property(_get, _set)
 
 
+@total_ordering
```

Comment:
If you're willing, in this case I think we would prefer writing the comparison methods ([trying to minimize dependencies](https://github.com/datastax/python-driver/blob/master/CONTRIBUTING.rst#design-and-implementation-guidelines)).